docs/src/content/docs/components/input-select.mdx

```diff
@@ -73,10 +79,61 @@ Displays a dropdown field, allowing a user to select an option from a pre-define
 				placeholder: 'Choose one...',
 				initialValue: [{ label: 'Option 1', value: 'option1' }]
 			}
+		},
+		{
+			typeName: 'display.text',
+			props: {
+				label: 'Selected label: {{selectInput.label}}'
+			}
+		},
+		{
+			typeName: 'display.text',
+			props: {
+				label: 'Selected value: {{selectInput.value}}'
+			}
 		}
 	]}
 />
 
+## Runtime State
+
+The value in the runtime state is the string value of the select input, using dot notation to
```

Comment:
The value stored in the runtime storage is not really a string. Is an object `{ label: string; value: string }`, right?